compiler/luci-interpreter/pal/linux/PALBroadcastTo.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LUCI_INTERPRETER_PAL_BROADCASTTO_H
+#define LUCI_INTERPRETER_PAL_BROADCASTTO_H
+
+#include <tensorflow/lite/kernels/internal/reference/broadcast_to.h>
+
+namespace luci_interpreter_pal
+{
+static inline void BroadcastTo(const tflite::RuntimeShape &input_shape, const char *input_data,
+                               const tflite::RuntimeShape &output_shape, char *output_data,
+                               TfLiteType data_type)
+{
+  constexpr int kMaxDims = 5;
```

Comment:
Can you add a comment about the value? Why `kMaxDims` is 5?

Reply:
@jinevening 

If the current `kMaxDims` is set to 8 specified in [tensorflow](https://github.com/tensorflow/tensorflow/blob/1b8f5c396f0c016ebe81fe1af029e6f205c926a4/tensorflow/lite/kernels/broadcast_to.cc#L34), 
memory out-of-bound error occurs in relation to `memcpy` in ubuntu 18.04. 
This has a compiler bug related to gcc version, so the value of `kMaxDims` seems to be supported up to 5.
In ubuntu 18.04, the basic version of gcc is 7.4, and it seems that there is a bug related to built_in_memcpy only in this version. This is because the upper versions, Ubuntu 20.04 and 22.04, have a basic version of gcc of 9.4 and no related errors occur here.
i posted a bug-related issue on [tensorflow](https://github.com/tensorflow/tensorflow/issues/62440),
but I didn't get the exact answer related to the bug. 😅
So I'm going to add the code set to `kMaxDims` as 5, and I'm going to add bug-related comments.